.vscode/launch.json

```diff
@@ -71,7 +71,7 @@
       "cwd": "${workspaceRoot}",
       "stopOnEntry": false,
       "args": [
-        "test/liveValidatorTests.js"
+        "test/modelValidatorTests.js"
```

Comment:
is this intentional, should it be one or the other?

Reply:
It could be any, doesnt matter.